reactRouter/list.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+const List = ({clientList}) => (
+  <ul>
+    {clientList.length &&clientList.map((item,index) => 
+      <li key={index}>{item.Name}</li>
```

Comment:
@VARTIKASINGH1234 What would occur, If a key is {0}?

Reply:
@VARTIKASINGH1234 I'm getting some warning messages while Checking the PR. Please resolved that warnings.